frontend/src/views/Camps.vue

```diff
@@ -44,15 +45,19 @@
 </template>
 
 <script>
-import { campRoute } from '@/router'
+import router, { campRoute } from '@/router'
```

Comment:
```suggestion
import { campRoute } from '@/router'
```
Rückgängig machen, da `router` nicht gebraucht wird wenn wir `goToCamp` nicht brauchen